### PR TITLE
Cleanup on client listener service

### PR DESCRIPTION
**Pull request by sancar**
Code is made more readable with keeping the behaviour as is.

### CODE REVIEW

**Comment by sancar** [comment]
run-lab-run

**Comment by sancar** [comment]
run-lab-run

**Comment by sancar** [comment]
run-lab-run

File: hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/listener/ClientListenerRegistration.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    public ListenerMessageCodec getCodec() {
+        return codec;
+    }
+
+    public EventHandler getHandler() {
+        return handler;
+    }
+
+    Map<Connection, ClientConnectionRegistration> getConnectionRegistrations() {
```
Comment (ihsandemir) [inline hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/listener/ClientListenerRegistration.java]: exposing a private variable to outside this class (even though the same package).

Alternative approach: Do not expose it outside and provide put, get, iterator() methods directly on `ClientListenerRegistration` class.

Comment (sancar) [reply hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/listener/ClientListenerRegistration.java]: One can argue same thing about other two fields also. I would like to think that `ClientListenerRegistration` is just a holder class and we do not do any operations `put` `get` etc.. on it.

**Comment by sancar** [comment]
run-lab-run

**Pull request merged**